cli/src/main/java/org/jboss/as/cli/handlers/CommandHandlerWithHelp.java

```diff
@@ -91,8 +91,8 @@ public void handle(CommandContext ctx) throws CommandLineException {
     protected void printHelp(CommandContext ctx) throws CommandLineException {
         InputStream helpInput = WildFlySecurityManager.getClassLoaderPrivileged(CommandHandlerWithHelp.class).getResourceAsStream(filename);
         if(helpInput != null) {
-            BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput));
-            try {
+
+            try (final BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput, StandardCharsets.UTF_8));){
```

Comment:
This is an example of what I am talking about. This change to try with resources is unrelated to the change to pass the charset to new InputStreamReader.

If we merge a PR with a lot of these kinds of changes I can almost guarantee that there will be bugs introduced and the hassle from them will be greater than whatever trouble we get from not passing in the charset. And I don't want to go through dozens of unrelated changes like this and try to analyze if they are safe.